LINAC.c

```diff
@@ -0,0 +1,33 @@
+#include <xc.h>
+#include "LINAC.h"
+
+ void LINAC_init() {
+    TRISB1 = 0;     //set MOTOR_FWD as output
+    LATB1 = 0;
+    
+    TRISB2 = 0;     //set MOTOR_BWD as output
+    LATB2 = 0;
+    
+    TRISB3 = 1;     //set LIM_HIGH as input
+    ANSELB3 = 0;    //set as digital input
+    
+    TRISB4 = 1;     //set LIM_LOW as input
+    ANSELB4 = 0;    //set as digital input
+    
+}
+ 
+int open_vent(){
```

Comment:
nitpick: the C language standard specifies that a function with no arguments should be declared as `open_vent(void)`. Refer to: https://wiki.sei.cmu.edu/confluence/display/c/DCL20-C.+Explicitly+specify+void+when+a+function+accepts+no+arguments 

xc8 also throws a warning in this case. make sure you read the output and you'll catch more things like this.

Note that this is not the case in C++.